### DISCUSSION TITLE
How do I do an island-detection

### DISCUSSION THREAD

**Discussion by Johneltonsbrain** [sn4k3/UVtools#208] [Q&A]
I sliced with Prusa Slocer for my SL1

**Comment by sn4k3** [comment]
By default, when you click the Issues tab (Yellow whell with ! red sign) it will auto compute issues for the first time on that file.
But if nothing trigger you can go to the same tab and click "Detect" to manually compute issues. If nothing shows then there are no issues.

![image](https://user-images.githubusercontent.com/113281/120129951-f1392a80-c1bc-11eb-8cc0-66a4dd32dd89.png)
